docs/sources/features/datasources/elasticsearch.md

```diff
@@ -117,7 +117,7 @@ Some metric aggregations are called Pipeline aggregations, for example, *Moving
 ## Templating
 
 Instead of hard-coding things like server, application and sensor name in you metric queries you can use variables in their place.
```

Comment:
```suggestion
Instead of hard-coding things like server, application and sensor name in your metric queries you can use variables in their place.
```